pandas/tests/groupby/test_groupby.py

```diff
@@ -75,6 +75,20 @@ def test_basic(dtype):
     with pytest.raises(Exception, match=msg):
         grouped.aggregate(lambda x: x * 2)
 
+    # issue 14873
+    dat = pd.DataFrame(
+        {'a': [1, 1, 2, 2], 'b': [0, 1, 2, 3], 'c': [None, None, 1, 1]})
+    gp = dat.groupby('a')
+    case_one_res = gp['b'].aggregate(lambda x: (x != 0).all())
+    case_two_res = gp['c'].aggregate(lambda x: x.isnull().all())
+
+    index = Index([1, 2], name='a')
+    case_one_exp = Series([False, True], index=index, name='b')
+    case_two_exp = Series([True, False], index=index, name='c')
+
+    assert_series_equal(case_one_res, case_one_exp)
+    assert_series_equal(case_two_res, case_two_exp)
+
```

Comment:
Can we just make this a separate test?

In general, we are trying to move toward smaller, more modular tests.

Reply:
@gfyoung created a separate test.